src/utils.js

```diff
@@ -314,10 +314,21 @@ export function optionsList(schema) {
 
 function findSchemaDefinition($ref, definitions = {}) {
   // Extract and use the referenced definition if we have it.
-  const match = /#\/definitions\/(.*)$/.exec($ref);
-  if (match && match[1] && definitions.hasOwnProperty(match[1])) {
-    return definitions[match[1]];
+  const match = /^#\/definitions\/(.*)$/.exec($ref);
+  if (match && match[1]) {
+    const parts = match[1].split("/");
```

Comment:
Makes me think of properties containing a `/` in their name, which will fail here. I don't want us to over this use case, but it may be a good idea to document this limitation, or at least adding a comment about it here.

Reply:
@epicfaace: Perhaps I had misunderstood what @glasserc was proposing a FIXME comment for—I assumed it was to handle `/`s, which I fixed in 95bee41 so comment no longer required?

Reply:
> @epicfaace: Perhaps I had misunderstood what @glasserc was proposing a FIXME comment for—I assumed it was to handle `/`s, which I fixed in [95bee41](https://github.com/mozilla-services/react-jsonschema-form/commit/95bee418412810e67fb18c8d1a6c59c16d900ff6) so comment no longer required?

Oh, I assumed that the FIXME comment was supposed to be for using "json-schema-ref-parser to handle ... other issues."

Since you did fix the escaping of `/` and `~`, do you know if there's anything we're missing in this implementation which json-schema-ref-parser does well?